README.md

```diff
@@ -1,27 +1,72 @@
 
 Use the ECS CLI to:
-* Bring your existing Docker apps
-* Set up staging and production environments, cross region and cross account
-* Set up production-ready, battle tested ECS Clusters, Services and infrastructure
-* Set up CI/CD Pipelines for all of the micro-services that make up your Project
-* Monitor and debug your applications
+* Organize all your related micro-services in one project
+* Set up test and production environments, across regions and accounts
+* Set up production-ready, scalable ECS services and infrastructure
+* Set up CI/CD Pipelines for all of the micro-services
+* Monitor and debug your applications from your terminal
 
 ![ecs-cli help](https://user-images.githubusercontent.com/828419/69765586-5c69f280-1129-11ea-9427-623d15975940.png)
 
-## 🌟 Tenets (unless you know better ones)
+## Installing
+
+During preview, we're distributing binaries from our GitHub releases. Instructions for installing the ECS CLI v2 for your platform:
+
+<details>
+  <summary>macOS and Linux</summary>
+
+
+| Platform | Command to install |
+|---------|---------
+| macOS | `curl -Lo /usr/local/bin/ecs-preview https://github.com/aws/amazon-ecs-cli-v2/releases/download/latest/ecs-preview-darwin-v0.0.5 && chmod +x /usr/local/bin/ecs-preview && ecs-preview --help` |
+| Linux | `curl -Lo /usr/local/bin/ecs-preview https://github.com/aws/amazon-ecs-cli-v2/releases/download/latest/ecs-preview-linux-v0.0.5 && chmod +x /usr/local/bin/ecs-preview && ecs-preview --help` |
+
+</details>
+
+
+## Getting started
+
+Make sure you have the AWS command line tool installed and have already run `aws configure` before you start.
+
+To get a sample app up and running in one command, run the following:
+
+```sh
+git clone git@github.com:aws-samples/amazon-ecs-cli-sample-app.git demo-app
+cd demo-app
+ecs-preview init --project demo      \
+  --app api                          \
+  --app-type 'Load Balanced Web App' \
+  --dockerfile './Dockerfile'        \
+  --deploy
+```
+
+This will create a VPC, Application Load Balancer, an Amazon ECS Service with the sample app running on AWS Fargate. This process will take around 8 minutes to complete - at which point you'll get a URL for your sample app running!
+
+<details>
+    <summary> watch what happens when you run <tt>init</tt></summary>
+
+![Step By Step Setup](https://user-images.githubusercontent.com/828419/69770895-91813f80-113f-11ea-8be9-60df6c2bf3fc.gif)
+</details>
+
+### Cleaning up 🧹
+
+Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `ecs-preview project delete`.
+
+### Learning more 📖
+
+Want to learn more about what's happening? Check out our [getting started guide](https://github.com/aws/amazon-ecs-cli-v2/wiki/Getting-Started) and a detailed breakdown of all the [infrastructure](https://github.com/aws/amazon-ecs-cli-v2/wiki/Infrastructure) that gets created.
+
+## Tenets 🌟
```

Comment:
Should we move this to a `CHARTER.md` like firecracker does? https://github.com/firecracker-microvm/firecracker/blob/master/CHARTER.md

Reply:
The Tenets section? I can do that :D